lib/functions/rootfs/distro-agnostic.sh

```
@@ -7,6 +7,70 @@
+		fi
+		CONSOLEARGS="${CONSOLEARGS:-}${CONSOLEARGS:+ }${ITEM:?}"
+	done
+
+	export BOOTSCRIPT_TEMPLATE__SERIAL_CONSOLE="${CONSOLEARGS:-}"
+}
+
+function render_bootscript_template() { (
+	typeset BOOTSCRIPT_TEMPLATE__CREATE_DATE
+	typeset SHELL_FORMAT
+
+	BOOTSCRIPT_TEMPLATE__CREATE_DATE="$(date -Ru)"
```

Shellcheck (0.10.0)

[warning] BOOTSCRIPT_TEMPLATE__CREATE_DATE appears unused. Verify use (or export if used externally).

(SC2034)